stack/run.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:noble
+
+ARG sources
+ARG packages
+ARG package_args='--no-install-recommends'
+
+RUN echo "$sources" > /etc/apt/sources.list.d/ubuntu.sources
+
+RUN echo "debconf debconf/frontend select noninteractive" | debconf-set-selections && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get -y $package_args update && \
+  apt-get -y $package_args upgrade && \
+  apt-get -y $package_args install locales && \
+  locale-gen en_US.UTF-8 && \
+  update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 && \
+  apt-get -y $package_args install $packages && \
+  find /usr/share/doc/*/* ! -name copyright | xargs rm -rf && \
+  rm -rf \
+    /usr/share/man/* /usr/share/info/* \
+    /usr/share/groff/* /usr/share/lintian/* /usr/share/linda/* \
+    /var/lib/apt/lists/* /tmp/*
```

Comment:
I noticed this file differs from Noble Base and Jammy Full in that its missing:
```
  echo "Package: $packages\nPin: release c=multiverse\nPin-Priority: -1\n\nPackage: $packages\nPin: release c=restricted\nPin-Priority: -1\n" > /etc/apt/preferences && \
  echo "debconf debconf/frontend select noninteractive" | debconf-set-selections && \ 
```
Is there a reason for this?

Reply:
no reason, wrong copy&paste 🙂

fixed